test/functional/api/v2/users_controller_test.rb

```diff
@@ -1,6 +1,7 @@
 require 'test_helper'
 
 class Api::V2::UsersControllerTest < ActionController::TestCase
+  AuthSourceInternal.create(:name => "Internal") unless AuthSourceInternal.first.present?
```

Comment:
true, but because controller tests are not wrapped in a transaction another test might delete all auth sources.